clang/include/clang/AST/FormatString.h

```diff
@@ -292,7 +292,7 @@ class ArgType {
   };
 
 private:
-  const Kind K;
+  Kind K;
```

Comment:
Oh? Is this necessary?

Reply:
The `const` field was the only thing that prevented `ArgType` from getting a copy constructor and assignment operators.